@xen-orchestra/lite/src/components/pool/PoolTabBar.vue

```diff
@@ -25,7 +25,7 @@
       {{ t('hosts') }}
     </RouterTab>
     <RouterTab :to="{ name: 'pool.vms', params: { uuid: pool?.uuid } }">
-      {{ t('vms') }}
+      {{ t('vms', 0) }}
```

Comment:
According to Figma, show the `vms` variant.
```suggestion
      {{ t('vms', 2) }}
```

Reply:
Well in tabs we don't have "many" VMs so i don't know if we should use like we got 2 VMs, instead of 0 VMs , as in vuei18n example : 
```
const messages = {
  en: {
    apple: 'no apples | one apple | {count} apples'
  }
}
```